css/symb/main.css

```diff
@@ -228,6 +228,10 @@ table.styledtable tr.alt td {
 	background:#999999; 
 }
 
+.welcome-text {
```

Comment:
This is the wrong main.css, I think. Move this to css/v202209/symbiota/main.css.

I know - there are too main main.csses haha.